pkg/registry/apis/provisioning/resources/parser.go

```diff
@@ -148,6 +148,7 @@ func (r *FileParser) Parse(ctx context.Context, logger *slog.Logger, info *repos
 			}
 		}
 		linter := lint.NewDashboardLinter()
+		// TODO: pass the config file
```

Comment:
Loading lint from the repo is interesting, but I think the path should be saved in the repo spec, not auto-detected